jfrog_platform_cloud/manifest.json

```diff
@@ -0,0 +1,54 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "798102cb-6c52-4a16-bc1b-48c2e6b54e71",
+  "app_id": "jfrog-platform-cloud",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View and analyze JFrog Artifactory Cloud logs",
+    "title": "JFrog Platform Cloud",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Logs dashboard, HTTP requests",
+        "image_url": "images/jfrog_platform_cloud_logs_0.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Logs dashboard, Request logs",
+        "image_url": "images/jfrog_platform_cloud_logs_1.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "JFrog Artifactory Logs dashboard, Operations",
+        "image_url": "images/jfrog_platform_cloud_logs_2.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Log Collection",
+      "Offering::Integration",
+      "Submitted Data Type::Logs",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "assets": {
+    "dashboards": {
+      "JFrog Platform Cloud Log Analytics\n": "assets/dashboards/jfrog_platform_cloud_overview.json"
+    },
+    "monitors": {},
+    "logs": {},
+    "saved_views": {}
```

Comment:
You will also need an `integration` object here. See [manifest file docs](https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file) and [Twingate](https://github.com/DataDog/integrations-extras/blob/master/twingate/manifest.json) as an example.

Reply:
Added.